code/game/turfs/simulated/floor/plating/asteroid.dm

```diff
@@ -68,7 +68,27 @@
 			if(W.use_tool(src, user, 40, volume=50))
 				if(!can_dig(user))
 					return TRUE
-				to_chat(user, "<span class='notice'>You dig a hole.</span>")
+
+				if(W.tool_behaviour == TOOL_MINING && prob(7)) //this may be too often - in testing 20 felt like too rare to be worth exploring
+					to_chat(user, "<span class='warning'>You opened a fissure! A gas is escaping...</span>")
+					icon_state = "fissure"
+					icon_plating = "fissure"
+					environment_type="fissure"
+					switch(rand(1,10)) //This probably needs to be adjusted - picks between different types of fissures
+						if(1,2)
+							initial_gas_mix = "plasma=32;n2=17;TEMP=360"
+						if(4,3,5)
+							initial_gas_mix = "co2=33;n2=7;TEMP=450"
+						if(6)
+							initial_gas_mix = "tritium=24;co2=20;no2=2;TEMP=360"
+						if(7)
+							initial_gas_mix = "nob=12;pluox=7;n2=32;water_vapor=70;TEMP=170"
+						if(8)
+							initial_gas_mix = "pluox=32;stim=13;TEMP=295"
```

Comment:
haha who needs fusion when you can dig while dragging a scrubber

Reply:
the gas mixes probably need work but I didn't know what I was doing so I played it by ear and tossed in what I thought might be fun to find. If you have suggestions on what the gas mixes should be I'm not committed to these

Reply:
anything that isnt nb or stimulum is fine

Reply:
yeah i reworked them, made the mixes a bit less rich but more things mixed together and also always above ignition except for the cold pluoxy fissure to make them a bit more dangerous to look for